src/menu/flower.cljs

```diff
@@ -20,8 +20,8 @@
  (let [transition-length (/ transition-length 2)]
   (h/div
    :css (j/cell= {:position "fixed"
-                  :left (u/n->px outer-radius)
-                  :bottom (u/n->px outer-radius)
+                  :left (u/n->px (.round js/Math outer-radius))
```

Comment:
why not define a new function like `(def n->px-rounded (comp n->px Math/round))`